client/src/js/views/DashboardDogsListDogView.js

```diff
@@ -7,26 +8,47 @@ var DashboardDogsListDogView = Backbone.View.extend({
   template: _.template(DashboardDogsListDogTemplate),
 
   events: {
-    'change [name="checked_in"]': 'handleCheckedInChange',
-    'change [type="radio"]' : 'statusChange'
+    'click .button--check-in': 'handleCheckedInChange',
+    'change [type="radio"]' : 'statusChange',
+    'click .dropdown' : 'toggleDropdown'
   },
 
   initialize: function() {
     this.render();
+    this.listenTo(this.model, 'change:current_status', this.render);
   },
-
+ 
   render: function() {
-    this.$el.html(this.template(this.model.attributes));
+
+    var verbose_status = this.model.setVerboseStatus();
```

Comment:
Not for model attributes, those are all using snake_case right now. If we want to camrlCase them we should change them all.